node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonConfig.java

```diff
@@ -99,13 +99,14 @@ public class CommonConfig {
    * ClientManager will have so many selector threads (TAsyncClientManager) to distribute to its
    * clients.
    */
-  private int selectorNumOfClientManager =
-      Runtime.getRuntime().availableProcessors() / 4 > 0
-          ? Runtime.getRuntime().availableProcessors() / 4
-          : 1;
+  private int selectorNumOfClientManager = 1;
 
   /** whether to use thrift compression. */
-  private boolean isCnRpcThriftCompressionEnabled = false;
+  private boolean isRpcThriftCompressionEnabled = false;
+
+  private int maxTotalClientForEachNode = 300;
+
+  private int maxIdleClientForEachNode = 200;
```

Comment:
These updated parameters should be also removed in ConfigNodeConfig, IoTDBConfig, iotdb-confignode.properties and iotdb-datanode.properties. And should be added in iotdb-commons.properties.

Reply:
Current thrift related parameters are placed in a separate configuration file for confignode/datanode. This ensures that confignode and datanode can use different thrift parameters in a distribution package.

To keep up with the current status, I load different parameters for the same variable in CommonDescriptor, which is ugly but ensures that any process can set the parameters correctly to initialize the clientPool with the correct configuration.